src/Features/CSharpTest/ExtractMethod/ExtractMethodTests2.cs

```diff
@@ -2,6 +2,7 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
+using System.Security;
```

Comment:
Was this intentional? It doesn't look to me like anything uses something from System.Security.

Reply:
nope!